unit_test/fuzzing/test_requester/test_spdm_requester_get_csr/get_csr.c

```diff
@@ -248,6 +248,64 @@ void libspdm_test_requester_get_csr_case2(void **State)
                     &csr_len);
 }
 
+void libspdm_test_requester_get_csr_case3(void **State)
+{
+    libspdm_test_context_t *spdm_test_context;
+    libspdm_context_t *spdm_context;
+    uint32_t session_id;
+    libspdm_session_info_t *session_info;
+
+    uint8_t csr_form_get[LIBSPDM_MAX_CSR_SIZE] = {0};
+    size_t csr_len;
+    csr_len = LIBSPDM_MAX_CSR_SIZE;
+
+    spdm_test_context = *State;
+    m_secured_on_off = true;
+    spdm_context = spdm_test_context->spdm_context;
+    spdm_context->connection_info.version = SPDM_MESSAGE_VERSION_12 <<
+                                            SPDM_VERSION_NUMBER_SHIFT_BIT;
+    spdm_context->connection_info.connection_state =
+        LIBSPDM_CONNECTION_STATE_AUTHENTICATED;
+    spdm_context->connection_info.capability.flags = 0;
+    spdm_context->connection_info.capability.flags |=
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_MEAS_CAP_SIG |
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_PSK_CAP |
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_ENCRYPT_CAP |
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_MAC_CAP |
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_CERT_INSTALL_RESET_CAP;
+    spdm_context->local_context.capability.flags = 0;
+    spdm_context->local_context.capability.flags |=
+        SPDM_GET_CAPABILITIES_REQUEST_FLAGS_PSK_CAP |
+        SPDM_GET_CAPABILITIES_REQUEST_FLAGS_MAC_CAP |
+        SPDM_GET_CAPABILITIES_REQUEST_FLAGS_ENCRYPT_CAP;
+
+    spdm_context->connection_info.algorithm.dhe_named_group =
+        m_libspdm_use_dhe_algo;
+    spdm_context->connection_info.algorithm.aead_cipher_suite =
+        m_libspdm_use_aead_algo;
+
+    spdm_context->connection_info.connection_state =
+        LIBSPDM_CONNECTION_STATE_NEGOTIATED;
+    spdm_context->connection_info.capability.flags |=
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_CSR_CAP;
+
+    session_id = 0xFFFFFFFF;
+    session_info = &spdm_context->session_info[0];
+    libspdm_session_info_init(spdm_context, session_info, session_id, true);
+    libspdm_secured_message_set_session_state(session_info->secured_message_context,
+                                              LIBSPDM_SESSION_STATE_ESTABLISHED);
+
+#if LIBSPDM_DEBUG_ASSERT_ENABLE
```

Comment:
Please use the cmock macros. For example https://github.com/DMTF/libspdm/blob/e2f08f1cac9aa66970be5bf1f6d4b125deacc054/unit_test/test_spdm_requester/get_version.c#L557-L558 Those are present regardless of the value of `LIBSPDM_DEBUG_ASSERT_ENABLE`.

Reply:
`assert_int_equal()` isn't exposed in the fuzzer, so that doesn't work. Do you also want me to add support for cmock macros to the fuzzer?

Reply:
Or I can drop the tests from the fuzzer, up to you